projects/plugins/jetpack/extensions/blocks/ai-chat/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 3,
 	"name": "jetpack/ai-chat",
-	"title": "AI Chat (Experimental)",
+	"title": "AI Chat (Beta)",
```

Comment:
I'm making that change to clearly mark it as Beta (vs. the other AI blocks currently marked as experimental).